src/data/veilarboppfolging/veilarboppfolgingTypes.ts

```diff
@@ -0,0 +1,8 @@
+export interface UnderArbeidsrettetOppfolgingRequestDTO {
+  fnr: string;
+}
+
+export interface UnderArbeidsrettetOppfolgingResponseDTO {
+  underOppfolging: boolean;
+  erManuell: boolean;
```

Comment:
Hva betyr dette?

Reply:
Vet ikke 🙃

Reply:
Gitt [svaret på slack](https://nav-it.slack.com/archives/C04HS60F283/p1720790328040349?thread_ts=1717674508.182319&cid=C04HS60F283) så kan vi kanskje bare fjerne at vi enkoder det fra responsen? :)